.github/CODEOWNERS

```
@@ -110,6 +110,7 @@ pkg/translator/prometheusremotewrite/                @open-telemetry/collector-c
 pkg/translator/signalfx/                             @open-telemetry/collector-contrib-approvers @pmcollins @pjanotti @dmitryax
 pkg/translator/zipkin/                               @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
 pkg/winperfcounters/                                 @open-telemetry/collector-contrib-approvers @dashpole @mrod1598 @binaryfissiongames
+pkg/batchperresourceattr/                            @open-telemetry/collector-contrib-approvers @atoulme @pmcollins @dmitryax @owais
```

```suggestion
pkg/batchperresourceattr/                            @open-telemetry/collector-contrib-approvers @atoulme @pmcollins @dmitryax
```
Owais is not actively involved in OTel now